src/sage/sets/set_from_iterator.py

```diff
@@ -460,7 +460,7 @@ def _instancedoc_(self):
             sage: print(sage_getdoc(d))   # indirect doctest
                Test whether "self" is prime.
             ...
-               Calls the PARI "isprime" function.
+               Calls the PARI ...isprime...
         """
```

Comment:
Why dots?

Reply:
It's formatted as a cross-reference to the PARI documentation, and I didn't want to tie the test to the specifics of how that looks

Reply:
You mean that you don't want to rely on how the role `:pari:` is actually rendered. OK.